md/release-notes.md

```diff
@@ -10,8 +10,10 @@
 <a id="data-management"/>
 
 ### Business Data Management in the UI Designer
-Introspect and visualize your Business Data model in your UI Designer pages with a few clicks to speed up your development
 
+Add sections to your pages and forms to display Business Data Information in master-detail representation in a few clicks and introspect and visualize Business Data Model in your browser to help you know and use your data model.
+
+More details [here](data-management.md).
```

Comment:
What about:
Visualize and explore your Business Data Model (BDM) in your browser.
In a few clicks, generate user interfaces from the BDM, using the master/details pattern.